pkg/server/structlogging/hot_ranges_log.go

```diff
@@ -25,7 +25,21 @@ import (
 )
 
 // ReportTopHottestRanges limits the number of ranges to be reported per iteration
-const ReportTopHottestRanges = 5
+var ReportTopHottestRanges int32 = 5
+
+// CheckInterval is the interval at which the system checks
+// whether or not to log the hot ranges.
+var CheckInterval = time.Second
+
+// TestLoopChannel triggers the hot ranges logging loop to start again.
+// It's useful in the context of a test, where we don't want to wait
+// for whatever the last time the interval was.
+var TestLoopChannel = make(chan struct{}, 1)
+
+// LogCPUThreshold defines the cpu duration per second
+// which needs to be exceeded for the system to automatically
+// log the hot ranges.
+var LogCPUThreshold = time.Second / 4
```

Comment:
Can you provide some context around how this was chosen (even if that is arbitrary).

Also, do you think this should be a cluster setting?

Reply:
It originally was meant to emulate [this existing cluster setting](https://github.com/angles-n-daemons/cockroach/blob/master/pkg/kv/kvserver/replica_split_load.go#L52), though we wanted the system to be slightly more sensitive.

I think making this a cluster setting is a good idea, I'll change that now.